R/utils.R

```diff
@@ -24,7 +24,8 @@ get_namespace_object_names <- function(ns) {
 #'
 #' @importFrom utils packageVersion
 new_empty_test_trace_tally <- function() {
-  if (utils::packageVersion("covr") < "3.6.3") {
+  # version needs to be updated until changes are merged
+  if (utils::packageVersion("covr") <= "3.6.4.9000") {
```

Comment:
This was the culprit for the unexpectedly failing tests. 

Since this PR is still hanging in the ether (and I need to close it and probably submit a new one to reinvigorate that discussion), the version-dependent data structures weren't matching up. Needed to bump the version so that the tests pass with the current version of `covr`.

Reply:
Ach, makes sense. Great catch with that one!